contracts/SimpleSwapFactory.sol

```diff
@@ -16,18 +17,23 @@ contract SimpleSwapFactory {
 
   /* address of the ERC20-token, to be used by the to-be-deployed chequebooks */
   address public ERC20Address;
+  /* address of the code contract from which all chequebooks are cloned */
+  address public master;
 
-  constructor(address _ERC20Address) public {
+  constructor(address _ERC20Address, address _master) public {
```

Comment:
I think it is cleaner if you deploy the master contract as part of the constructor of the factory

Reply:
that is a good idea, that also solves the master initialisation problem. will do that.

Reply:
done